vip-api/src/main/java/fr/insalyon/creatis/vip/api/controller/OidcLoginController.java

```diff
@@ -65,9 +67,15 @@ public RedirectView getOauth2LoginInfo(
 
         try {
             User vipUser = configurationBusiness.getOrCreateUser((String) userAttributes.get("email"), domainName, null);
+            Session session = new Session();
+
+            session.id = vipUser.getSession();
+            session.email = vipUser.getEmail();
+            
             SecurityContextHolder.clearContext(); // destroy spring session and use VIP own session mechanism
-            vipSessionBusiness.setVIPSession(request, response, vipUser); // creates VIP cookies and session
```

Comment:
We just lost the call to `configurationBusiness.updateUserLastLogin(user.getEmail());` in the process.
Not sure where to put it although.